docker/app.entrypoint.sh

```diff
@@ -1,52 +1,7 @@
 #!/bin/bash
 
-if [ "$USING_XDEBUG" == "1"  ]; then
-    echo "Enabling XDebug 3"
-    mv /usr/local/etc/php/conf.d/disabled/docker-php-ext-xdebug.ini /usr/local/etc/php/conf.d/
-fi
-
-# Run WordPress docker entrypoint.
-. docker-entrypoint.sh 'apache2'
-
-set +u
-
-# Ensure mysql is loaded
-dockerize -wait tcp://${DB_HOST}:${DB_HOST_PORT:-3306} -timeout 1m
-
-# Config WordPress
-if [ ! -f "${WP_ROOT_FOLDER}/wp-config.php" ]; then
-    wp config create \
-        --path="${WP_ROOT_FOLDER}" \
-        --dbname="${DB_NAME}" \
-        --dbuser="${DB_USER}" \
-        --dbpass="${DB_PASSWORD}" \
-        --dbhost="${DB_HOST}" \
-        --dbprefix="${WP_TABLE_PREFIX}" \
-        --skip-check \
-        --quiet \
-        --allow-root
-fi
-
-# Install WP if not yet installed
-if ! $( wp core is-installed --allow-root ); then
-	wp core install \
-		--path="${WP_ROOT_FOLDER}" \
-		--url="${WP_URL}" \
-		--title='Test' \
-		--admin_user="${ADMIN_USERNAME}" \
-		--admin_password="${ADMIN_PASSWORD}" \
-		--admin_email="${ADMIN_EMAIL}" \
-		--allow-root
-fi
-
-echo "Running WordPress version: $(wp core version --allow-root) at $(wp option get home --allow-root)"
-
-# Activate wp-graphql
-wp plugin activate wp-graphql --allow-root
-
-# Set pretty permalinks.
-wp rewrite structure '/%year%/%monthnum%/%postname%/' --allow-root
-
-wp db export "${PROJECT_DIR}/tests/_data/dump.sql" --allow-root
+# Run app setup script.
+. app-setup.sh
+. app-post-setup.sh
```

Comment:
My thinking with two scripts is that it would allow an inherited dockerfile entrypoint script to run the setup.sh, do some other stuff, then run the post-setup.sh.  Like install wp-graphl from somewhere if it's not installed, like how wp-graphql-acf needs.